MesaModel/model/Pupil.py

```diff
@@ -113,7 +113,7 @@ def redStateCange(self):
             self.time_in_green_state = 0
             return 1
 
-        # If the pupil cohort is inattentive and the teacher cohort is of
+        # If the pupils' inattentive switch is on and the teacher cohort is of
```

Comment:
Should we call it the model's inattentive switch? It's a parameter that the user sets on the whole model. We just class it as a pupil param because it affects the Pupil step.